engines/gob/degob_script_v7.cpp

```diff
@@ -0,0 +1,786 @@
+/* ScummVM Tools
+ *
+ * ScummVM Tools is the legal property of its developers, whose
+ * names are too numerous to list here. Please refer to the
+ * COPYRIGHT file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
```

Comment:
This is incorrect license. Please change to GPLv3+

Reply:
Now fixed.